### PR TITLE
CBI-653: Fetch correct metrics, aggregate and dump using debug

### PR DESCRIPTION
**Pull request by evgeny-s**

### CODE REVIEW

File: frame/example-offchain-worker/src/lib.rs [hunk 1]
```diff
@@ -268,71 +328,155 @@ impl<T: Trait> Module<T> {
...
 				http::Error::Unknown
 			})?;
 
-			// Print out our fetched JSON string
-			debug::info!("Node data: {}", resp_str);
+			let metrics_url_with_partition = format!("{}{}{}", metrics_url_with_key, METRICS_PARAM_PARTITIONID, partition_id_str);
+			// debug::info!("metrics_url_with_partition: {}", metrics_url_with_partition);
 
-			// Send signed Tx, TODO: send to call SC method.
+			let last_time_stamp: u64 = 1619690465923;
```
Comment (naure) [inline frame/example-offchain-worker/src/lib.rs]: For this ticket, just use `now - 24hours`

Comment (huythanhcere) [reply frame/example-offchain-worker/src/lib.rs]: Fixed. @naure

**Review by evgeny-s — COMMENTED** [review]
This method: `fetch_ddc_network_topology` is really huge. 
Can we separate it a little bit?

File: frame/example-offchain-worker/src/lib.rs [hunk 2]
```diff
@@ -52,8 +54,60 @@ pub const REPORT_METRICS_SELECTOR: [u8; 4] = hex!("35320bbe");
...
+	partitionId: Vec<u8>,
+	bytes: u32,
+	requests: u32,
+}
+
+impl MetricInfo {
+    fn new() -> Self {
+        Self {
+            appPubKey: Default::default(),
+			partitionId: Default::default(),
```
Comment (naure) [inline frame/example-offchain-worker/src/lib.rs:106]: Please use rustfmt. For instance with the Rust plugin in VSCode or Intellij.

**Pull request merged**